.github/scripts/check-forked-repos.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+if grep -q "github.com/.*rancher/shepherd =>" go.mod; then
```

Comment:
guessing we are capturing the replace directive, shouldn't the pattern just be `github.com/rancher/shepherd =>`

Reply:
sure I can rm the * check